test/JSONFileReaderWrongRowNumberTestFile.json

```diff
@@ -6,6 +6,8 @@
     [2]
   ],
   "cols": [
-    [1, 1]
+    [1, 1],
```

Comment:
This change is not associated with this task. It shouldn't be there.

Reply:
Codecov checks force this change. After one of the commits Codecov said that coverage has dropped by 0,03% (that is less then one line) and forced reparing JSON test in order to have all checks passed before merge.